lib/cube_browser/__init__.py

```diff
@@ -431,7 +431,7 @@ class Contourf(Plot2D):
     """
     def draw(self, cube):
         self.element = iplt.contourf(cube, axes=self.axes, coords=self.coords,
-                                     **self.kwargs)
+                                     extend='both', **self.kwargs)
```

Comment:
Hi @bjlittle - i totally agree about papering over. We had a chat to @marqh and decided that this quick fix was worth PRing (you may disagree of course!)